### ISSUE TITLE
Training loss isnt decreasing 

### ISSUE THREAD

**Issue by Mukil07** [Twizwei/bddoia_project#12]
Im using the faster_rcnn pretrained on BDD100k, and initiated its weight in the model_root parameter. I have set the freeze parameter to True.  

I have modified these lines of code in train.py 

from line 101, 
                `result = model(imBatch)`
                `pred=result[0]`
                `pred_reason=result[1]`
                
Please help me figure this out. Accumulated training loss remains constant
